ws-butler.el

```diff
@@ -78,6 +78,19 @@ i.e. only the \"virtual\" space is preserved in the buffer."
   :type 'boolean
   :group 'ws-butler)
 
+(defcustom ws-butler-convert-leading-tabs-or-spaces
+  t
+  "Make leading whitespace be tabs or spaces
+
+If `indent-tabs-mode' is non-nil, call `tabify', else call
+`untabify'. Do neither if `smart-tabs-mode' is enabled for this
+buffer. This is off by default because it's unwanted if you
```

Comment:
It appears to be on by default. Not sure whether you want to change the doc or the code...